reference/5.1/Microsoft.PowerShell.Core/About/about_CimSession.md

```diff
@@ -1,65 +1,62 @@
 ---
 keywords: powershell,cmdlet
 locale: en-us
-ms.date: 04/28/2018
+ms.date: 04/30/2020
 online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_cimsession?view=powershell-5.1&WT.mc_id=ps-gethelp
 schema: 2.0.0
 title: about_CimSession
 ---
-
 # About CimSession
 
-## SHORT DESCRIPTION
-
+## Short description
 Describes a CimSession object and the difference between CIM sessions and
 PowerShell sessions.
 
-## LONG DESCRIPTION
+## Long description
 
 A Common Information Model (CIM) session is a client-side object that
-represents a connection to a local computer or a remote computer. You can
-use CIM sessions as an alternative to PowerShell sessions
-(PSSessions). Both approaches have advantages.
+represents a connection to a local computer or a remote computer. You can use
+CIM sessions as an alternative to PowerShell sessions (PSSessions). Both
+approaches have advantages.
 
-You can use the New-CimSession cmdlet to create a CIM session that contains
-information about a connection, such as computer name, the protocol used
-for the connection, session ID, and instance ID.
+You can use the `New-CimSession` cmdlet to create a CIM session that contains
+information about a connection, such as computer name, the protocol used for
+the connection, session ID, and instance ID.
 
 After you create a CimSession object that specifies information required to
-establish a connection, PowerShell does not establish the
-connection immediately. When a cmdlet uses the CIM session,
-PowerShell connects to the specified computer, and then, when the
-cmdlet finishes, PowerShell terminates the connection.
+establish a connection, PowerShell does not establish the connection
+immediately. When a cmdlet uses the CIM session, PowerShell connects to the
+specified computer, and then, when the cmdlet finishes, PowerShell terminates
+the connection.
 
-If you create a PSSession instead of using a CIM session,
-PowerShell validates connection settings, and then establishes and
-maintains the connection. If you use CIM sessions, PowerShell
-does not open a network connection until needed. For more information about
-PowerShell sessions, see [about_PSSessions](about_PSSessions.md).
+If you create a PSSession instead of using a CIM session, PowerShell validates
```

Comment:
Should PSSESSION be bolded here since it is in the rest of the article?

Reply:
Fixed